MAUI/Autocomplete/UI-Customization.md

```diff
@@ -670,3 +670,27 @@ Completed event can be subscribed in C# also:
 The following image illustrates the result of the above code:
 
 ![.NET MAUI Autocomplete completed event](Images/UICustomization/CompletedEvent.png)
+
+## CursorPosition
+
+The cursor position in the input view can either be obtained or updated using the `CursorPosition` property in SfAutocomplete.
```

Comment:
The cursor position in the input view can be obtained or updated using the `CursorPosition` property in the SfAutocomplete.

Reply:
Hi @Chrispineagunja,

Changed the content